### PR TITLE
Make wrapper be /bin/steam

### PR DESCRIPTION
**Pull request by nanonyme**
As suggested by @smcv in #1078 let's make /bin/steam be wrapper script. This decreases probability of users accidentally bypassing wrapper and getting unexpected end results and facilicates some future use cases.

### CODE REVIEW

File: com.valvesoftware.Steam.yml [hunk 1]
```diff
@@ -180,13 +180,12 @@ modules:
         sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop
         sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop
         desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
-        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
-        desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
```
Comment (doraskayo) [inline com.valvesoftware.Steam.yml:184]: This looks unrelated.

File: com.valvesoftware.Steam.yml [hunk 2]
```diff
@@ -180,13 +180,12 @@ modules:
         sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop
         sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop
```
Comment (doraskayo) [inline com.valvesoftware.Steam.yml:181]: Shouldn't this be something like this?
```suggestion
        sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
```

File: com.valvesoftware.Steam.yml [hunk 3]
```diff
@@ -180,13 +180,12 @@ modules:
...
         desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
-        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
-        desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
         sed -i s:/usr/lib/:/app/lib/: steam
         PREFIX=/app make install
         ln -sf /bin/true /app/bin/steamdeps
         cp /usr/bin/cmp /app/bin
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
+        ln -sf /app/bin/steam-wrapper /app/bin/steam
```
Comment (doraskayo) [inline com.valvesoftware.Steam.yml:188]: I feel that `/app/bin/steam` originally being a symlink to `/app/lib/steam/bin_steam.sh` is Steam's own implementation detail and can change at any time. For example, it could become a script or even a binary file in any update.

With that in mind, it's safer to move `/app/bin/steam` someplace else instead of assuming that it's a symlink that we can remove.

File: steam_wrapper/steam_wrapper.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 
 FLATPAK_ID = os.getenv("FLATPAK_ID", "com.valvesoftware.Steam")
-STEAM_PATH = "/app/bin/steam"
+STEAM_PATH = "/app/lib/steam/bin_steam.sh"
```
Comment (doraskayo) [inline steam_wrapper/steam_wrapper.py:18]: If you follow my suggestion above, this should point at where we moved the original `/app/bin/steam`.

File: com.valvesoftware.Steam.yml (hunk 2 above)
Comment (doraskayo) [reply com.valvesoftware.Steam.yml:181]: Also, `command: steam-wrapper` should be `command: steam`.

Comment (nanonyme) [reply com.valvesoftware.Steam.yml:181]: I suppose you're right. @smcv any idea why it uses full path to steam in Desktop Entry?

File: com.valvesoftware.Steam.yml (hunk 3 above)
Comment (nanonyme) [reply com.valvesoftware.Steam.yml:188]: I don't think adding symlink buys anything. We could do test -x /app/lib/steam/bin_steam.sh to see the file exists in manifest so it fails in build time if this changes. Note that this path points to /app, it cannot arbitrarily change by without app rebuild.

Comment (nanonyme) [reply com.valvesoftware.Steam.yml:188]: Also, the path was suggested by @smcv, I am assuming they wouldn't have suggested it if it was likely to change.

File: com.valvesoftware.Steam.yml [hunk 5]
```diff
@@ -177,16 +177,16 @@ modules:
...
+        sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
         desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
+        desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
         sed -i s:/usr/lib/:/app/lib/: steam
         PREFIX=/app make install
         ln -sf /bin/true /app/bin/steamdeps
         cp /usr/bin/cmp /app/bin
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
+        ln -sf /app/bin/steam-wrapper /app/bin/steam
+        test -x
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:189]: A leftover?

File: com.valvesoftware.Steam.yml [hunk 6]
```diff
@@ -177,16 +177,16 @@ modules:
...
-        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
+        sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
         desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
+        desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
         sed -i s:/usr/lib/:/app/lib/: steam
         PREFIX=/app make install
         ln -sf /bin/true /app/bin/steamdeps
         cp /usr/bin/cmp /app/bin
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
+        ln -sf /app/bin/steam-wrapper /app/bin/steam
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:188]:
```suggestion
        ln -sf steam-wrapper /app/bin/steam
```

File: com.valvesoftware.Steam.yml (hunk 5 above)
Comment (nanonyme) [reply com.valvesoftware.Steam.yml:189]: This is actually missing the actual path. Will fix.
